src/app/property-management/property-management.component.html

```diff
@@ -0,0 +1,25 @@
+<div id="container">
+    <h3 id="page-title">Property Management</h3>
+    <section id="properties">
+        <button class="btn btn-primary" id="button-add-property" type="button">+ Add Property</button>
```

Comment:
Instead of using the character "+" we could use an icon inside : https://icons.getbootstrap.com/icons/plus/